src/common/snippets/src/lowered/pass/assign_registers.cpp

```diff
@@ -175,7 +175,8 @@ bool AssignRegisters::run(LinearIR& linear_ir) {
         std::map<Reg, Reg> register_map;
         std::stack<Reg> bank;
         // regs are stored in ascending order in reg_pool, so walk in reverse to assign them the same way
-        for (auto rit = reg_pool.crbegin(); rit != reg_pool.crend(); rit++) {
+        // NOLINTNEXTLINE(modernize-loop-convert)
```

Comment:
Why is required?

Reply:
It suggests replacing this loop with cpp20 ranges. I'd address that separately as well.